src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -33,6 +38,12 @@
     @Autowired
     private IdamClient idamClient;
 
+    @Autowired
+    private CaseDetailsConverter caseDetailsConverter;
+
+    @Autowired
+    private EmployeeObjectMapper employeeObjectMapper;
```

Comment:
Autowire for service classes is not ideal when coming to testing.  Better injecting via constructor

Reply:
Removed auto-wired and moved them as plain constructors.